src/formats/PDB.cpp

```diff
@@ -117,6 +142,22 @@ void PDBFormat::read(Frame& frame) {
             // Else we have read a frame
             got_end = true;
             continue;
+        case Record::HELIX:
+            read_HELIX(line);
+            continue;
+        case Record::SHEET:
+            read_secondary(line, 21, 32, "extended");
+            continue;
+        case Record::TURN:
+            read_secondary(line, 19, 30, "turn");
+            continue;
+        case Record::TER:
+            if (line.size() >= 12) {
+                try {
+                    atom_offsets_.push_back(std::stoul(line.substr(6,5)));
+                } catch(std::invalid_argument&) {} // Do nothing
```

Comment:
Should this send a warning?

Reply:
Originally I thought not, but actually, it makes sense as we should inform the user the line was ignored.